CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Extending the adopted spec, each change should have a link to its corresponding pull request appended.
 
+## [v4.1.0] - 2019-09-06
+
+### Added
+
+- Support for Forseti v2.20.0 [#246]
```

Comment:
Can you add in the actual links after line 245 and line 247, like so:

`[v4.1.0]: https://github.com/terraform-google-modules/terraform-google-forseti/compare/v4.0.1...v4.1.0`

`[#246]: https://github.com/forseti-security/terraform-google-forseti/pull/246`

And if there were other PRs that were merged in and not already listed, should add them here as well.